cores/esp32/esp32-hal-misc.c

```diff
@@ -54,6 +54,12 @@ void yield()
 extern TaskHandle_t loopTaskHandle;
 extern bool loopTaskWDTEnabled;
 
+void IRAM_ATTR abortLoopDelay(){
+    if(loopTaskHandle != NULL){
+        vTaskNotifyGiveFromISR(loopTaskHandle, NULL);
```

Comment:
This is an ISR function and is not usable outside the ISR context, you should check the caller context to see if it needs ISR calling or can use the non-ISR call. You can do this via:
```
if (xPortInIsrContext()) {
  vTaskNotifyGiveFromISR(....);
} else {
  vTaskNotifyGive(....);
}
```

Reply:
The API documentation isn't specific on this, thanks for the hint, done.